server/service/bookings.py

```diff
@@ -0,0 +1,62 @@
+
+from datetime import datetime
+import sys
+from bson.objectid import ObjectId
+
+class Bookings:
```

Comment:
No need of this class, forming the object logic can be moved to `booking_service.py`.
Create a utility class and move this methods  `get_no_of_days, get_add_ons_cost`  there